packages/ecr-credential-provider/ecr-credential-provider.spec

```diff
@@ -0,0 +1,49 @@
+%global goproject github.com/kubernetes
+%global gorepo cloud-provider-aws
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.24.1
+%global rpmver %{gover}
+
+%global _dwz_low_mem_die_limit 0
+
+%global gitrev b9c8d82db9edc3be45f82f8d5320d006c62aa50c
+%global shortrev %(c=%{gitrev}; echo ${c:0:7})
+
+Name: %{_cross_os}aws-ecr-credential-provider
+Version: %{rpmver}
+Release: 1%{?dist}
+Summary: Container image registry credential provider for AWS ECR
+License: Apache-2.0
+URL: https://github.com/kubernetes/cloud-provider-aws
+
+Source: v%{gover}.tar.gz
+Source1: bundled-v%{gover}.tar.gz
+Source1000: clarify.toml
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%prep
+%setup -n %{gorepo}-%{gitrev} -q
+%setup -T -D -n %{gorepo}-%{gitrev} -b 1 -q
+
+%build
+%set_cross_go_flags
+
+# Set CGO_ENABLED=0 to statically link ecr-credential-provider.
+go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o=ecr-credential-provider cmd/ecr-credential-provider/*.go
```

Comment:
I think that works for `hotdog` since there is only the `main.go` file in those directories. For this project there are multiple files that need to be compiled, and `go build` isn't happy if we just point to the parent directory.